README.md

```diff
@@ -27,6 +27,7 @@ page on the EFIDroid wiki for an exact mapping of LK targets to SoCs.
 - Asus Zenfone 2 Laser (720p) - Z00L
 - Asus Zenfone 2 Laser (1080p) - Z00T
 - Asus Zenfone Max ZC550KL (2016) - Z010D
+- BQ Aquaris M5 - piccolo
```

Comment:
Can you add a comment here that `lk2nd-msm8916-appended-dtb.img` should be used for this device?